tests/stdlib/tmacros.nim

```diff
@@ -0,0 +1,9 @@
+import macros
+
+block: # hasArgOfName
+  macro m(u:untyped): untyped =
+    for name in ["s","i","j","k","b","xs","ys"]:
+      doAssert hasArgOfName(params u,name)
+    doAssert not hasArgOfName(params u,"nonexistent")
+
+  proc p(s:string; i,j,k:int; b:bool; xs,ys:seq[int] = @[]) {.m.} = discard
```

Comment:
ditto

Reply:
I'll fix it here. Is there a formatting tool that automates this?

Reply:
`nimpretty` but you still need to double check results (and should only be applied on code you're changing, refs https://github.com/nim-lang/Nim/issues/8458#issuecomment-457055195

Reply:
It really doesn't matter for tests.